proto/asgt/v2/dataset_service.proto

```diff
@@ -95,88 +95,88 @@ service DatasetService {
 
 message GetDatasetRequest {
   string dataset_name = 1 [(validate.rules).string = {
-    pattern: "^[A-Za-z0-9.][A-Za-z0-9_.>-]*$",
+    pattern: "^[A-Za-z0-9.][A-Za-z0-9_.>-]*$"
```

Comment:
are the commas not needed?

Reply:
seems like you have a linter cleaning up things... we should align on that somehow

Reply:
latest version of buf i guess?